src/_variables.scss

```diff
@@ -16,6 +16,19 @@ $wb-assets-path: "#{$wb-core-path}/assets" !default;
 // BOOTSTRAP VARIABLES
 //
 
+//== Breadcrumb
+//
+//## Breacrumb enable/disable and colour settings.
+
+// WET Custom
+$wb-bc-enable: true !default; // Enable or disable WET's breadcrumb SCSS. Themes that don't use breadcrumbs can set this to false.
+$wb-bc-background-color: null !default; // Outer background colour. Override $breadcrumb-bg (Bootstrap variable) to change the inner background colour.
+$wb-bc-separator-color: #333 !default; // Separator colour.
+$wb-bc-separator-font-family: "Glyphicons Halflings" !default; // Separator font family.
+$wb-bc-separator-font-size: 0.7em !default; // Separator font size.
+$wb-bc-separator-content: "\e092" !default; // Separator content.
+$wb-bc-separator-content-rtl: "\e091" !default; // Separator content (right to left).
+
```

Comment:
@LaurentGoderre Done :). Btw just out of curiosity, what was the reasoning behind your suggestion?

When I initially placed the breadcrumb variables before the colour ones, I was thinking of putting things in alphabetical order. Felt kind of strange though, since it came right after the start of the Boostrap variables block. But on the other hand, that block already contained a mix of Bootstrap overrides and WET-specific variables, so it didn't feel any weirder than what was already being done.